services/madoc-ts/src/frontend/site/pages/tasks/review-listing/review-listing-page.tsx

```diff
@@ -116,62 +169,91 @@ export function ReviewListingPage() {
   // 5. Add alternative version with form and then actions, with a toggle.
 
   return (
-    <RefetchProvider refetch={refetch}>
```

Comment:
This may still be required, is there a refetch from `useInfiniteData` that could be passed down? This ensures the list reflects changes when you "approve" a task